components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3622,8 +3629,6 @@ subroutine compute_rain_fall_velocity(qr_incld, rcldm, rhofacr, nr, nr_incld, mu
 
    !Compute Vq, Vn:
 
-   nr  = max(nr,nsmall)
```

Comment:
Another minor issue I found with the p3 reference. There is no point in setting nr here since it is set again 7lines below and not used in the interim.